doc/generated/libpmemfile-posix.3

```diff
@@ -36,13 +36,10 @@
 \f[B]libpmemfile\-posix\f[] \- user space persistent memory aware file
 system API
 .SH SYNOPSIS
-.IP
-.nf
-\f[C]
-#include\ <libpmemfile\-posix.h>
-cc\ \-std=gnu99\ ..\ \-lpmemfile\-posix\ \-lpmemobj
-\f[]
-.fi
+.PP
+\f[C]#include\ <libpmemfile\-posix.h>\f[]
+.PP
+\f[C]cc\ \-std=gnu99\ ..\ `pkg\-config\ libpmemfile\-posix\ \-\-cflags\ \-\-libs`\f[]
```

Comment:
I disagree, let's go that far.
It can be used from C89, and C++ as well, so let's just remove the `-std` flag from here, and be thats it.